app/mixins/logging-model.js

```diff
@@ -22,13 +22,25 @@ export default Mixin.create({
       'splunk',
       'syslog',
       'fluentForwarder',
+      'customTarget',
     ];
 
     loggingTagets.forEach((key) => {
       let config
 
       if (key === 'fluentForwarder') {
         config = store.createRecord({ type: `fluentForwarderConfig`, });
+      } else if (key === 'customTarget') {
+        config = store.createRecord({ type: `customTargetConfig`, });
+        setProperties(config, {
+          clientKey:   '',
+          clientCert:  '',
+          certificate: '',
+          content:     `<match *>
```

Comment:
Fix indentation

Reply:
The `content` indentation is to display an initial file. I think it just like the log preview. 
![image](https://user-images.githubusercontent.com/18737885/51220234-59ca3880-196f-11e9-863a-676042f22f84.png)